schemamd/render.go

```diff
@@ -59,6 +59,10 @@ func writeAttribute(w io.Writer, path []string, att *tfjson.SchemaAttribute) ([]
 		return nil, err
 	}
 
+	if name == "id" {
```

Comment:
Maybe make sure its empty first also:

```suggestion
	if name == "id" && att.Description == "" {
```